dot/state/helpers_test.go

```diff
@@ -26,34 +26,25 @@ func newTriesEmpty() *Tries {
 	}
 }
 
-// newGenerator creates a new PRNG seeded with the
-// unix nanoseconds value of the current time.
-func newGenerator() (prng *rand.Rand) {
```

Comment:
So the new lib takes care of generating the seed and generator for us?

Reply:
yes